### PR TITLE
Overview section text

### PR DESCRIPTION
**Pull request by RCode-Blue**
## Overview of the Pull Request:
Edited wording for Overview section for legibility and clarity

## How Has This Been Tested?
The text section for the Overview section should look like this:

image

## Pull Request Checklist:
Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!
- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [X] Is your pull request up-to-date with `main` branch?
- [X] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [X] Have you written instructions on how to test that your changes work as expected?
- [X] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: components/ResultsPageComponents/SurveyOverview/SurveyOverview.jsx [hunk 1]
```diff
@@ -74,18 +75,21 @@ export default function SurveyOverview({
         Overview
       </Text>
       <Text fontWeight={400} fontSize="20px">
-        The Work Commute survey recorded results from 12 to 27 July 2022, with
-        responses from 162 people.
+        The Work Commute survey was conducted on 13 to 29 July 2022 and recorded
```
Comment (hqtan) [inline components/ResultsPageComponents/SurveyOverview/SurveyOverview.jsx:78]: we should align the survey date with what is in the infographic.

image

Can we double check the dates with Aainaa @RCode-Blue ?

Comment (RCode-Blue) [reply components/ResultsPageComponents/SurveyOverview/SurveyOverview.jsx:78]: Design team has confirmed that the dates are from 13th to 29th.

**Comment by hqtan** [comment]
Thx for confirming the survey start and end dates @RCode-Blue !

Can I trouble you to make another commit to update [the survey end date in the results JSON file please](https://github.com/CodeforAustralia/council-emissions-calculator/blob/main/data/2022-results.json#L3)?

**Review by hqtan — APPROVED** [review]
lgtm @RCode-Blue !
let's ship it! :shipit: 

Thanks for your efforts! 🙇🏽

**Pull request merged**